man/special-symbols.Rd

```diff
@@ -66,5 +68,25 @@ DT[{cat(sprintf('in i, .N is \%d\n', .N)); a < .N/2},
 # .I can be different in j and by, enabling rowwise operations in by
 DT[, .(.I, min(.SD[,-1]))]
 DT[, .(min(.SD[,-1])), by=.I]
+
+# These examples illustrate the importance of using copy(.BY) when capturing .BY values in a list outside the data.table context to avoid unexpected behavior.
+dt <- data.table(l = letters[1:5], b = rnorm(5))
```

Comment:
again please use simpler data (1:4 instead of rnorm) as I mentioned in my previous comment